e2e_test/schema_registry/alter_sr.slt

```diff
@@ -0,0 +1,74 @@
+# Before running this test, seed data into kafka:
+#   python3 e2e_test/schema_registry/pb.py <brokerlist> <schema-registry-url> <topic> <num-records>
+
+statement ok
+CREATE SOURCE src_user WITH (
+    connector = 'kafka',
+    topic = 'sr_pb_test',
+    properties.bootstrap.server = 'message_queue:29092',
+    scan.startup.mode = 'earliest'
+)
+FORMAT PLAIN ENCODE PROTOBUF(
+    schema.registry = 'http://message_queue:8081',
+    message = 'test.User'
```

Comment:
IIUC, the Conflulent Schema Registry supports versioning of schema, that is, one schema (topic) may have multiple schema with different versions. For example: https://docs.confluent.io/platform/current/schema-registry/develop/using.html#fetch-version-1-of-the-schema-registered-under-subject-kafka-value. Shall we support specifying an optional `version` here?

Reply:
I think this feature is not hard to implement. I will try it in the next pr.

Reply:
I think we can always stick to the latest schema version in source.
Here we assume the upstream workers always use the latest schema version (writer schema). It is not a strong assumption because if no producer uses a new schema, the schema version on the Schema Registry will not be updated. 
Schema registry guarantee backward compatibility, RW can get all it needs if `reader schema version <= writer schema version`, ie. there must be some producers generate data with equal or higher version of schema.

Reply:
That makes sense to me. We may defer it until being requested by some users.